### PR TITLE
Create template for PRs

### LINKED ISSUE #3: Create a MD template for the PRs descriptions
## Summary
- Create a markdown file that holds a template with required and important details to fill in when proposing a PR
- Feel free to propose a template
- This template should be just a guide don't add to much text with the headers is ok

### PR DESCRIPTION
**Pull request by diegoTech14**
# 📝 Create template for PRs

## 🛠️ Issue
- Closes #3

## 📖 Description
I added a template for Pull Requests, this improves the way PR is documented.

## ✅ Changes made
For this issue I've added the following sections:
1. Title PR section
2. Issue reference section
3. Issue Description section
4. Changes made section
5. Additional notes section

## 📜 Additional Notes
No additional notes for this PR

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# 📝 Pull Request Title
+
+## 🛠️ Issue
+- Closes #issue-ID
+
+## 📖 Description
+- Description of the issue
+
```
Comment (danielcdz) [inline .github/PULL_REQUEST_TEMPLATE.md:8]: Add section to fill the changes made

Comment (diegoTech14) [reply .github/PULL_REQUEST_TEMPLATE.md:8]: Got it sir, just the title and a bullet point right?

Comment (danielcdz) [reply .github/PULL_REQUEST_TEMPLATE.md:8]: yes sir

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+## 📜 Additional Notes
+- Some additional notes
+
+## ✅ Changes made
```
Comment (danielcdz) [inline .github/PULL_REQUEST_TEMPLATE.md:9]: Leave the additional notes as the last ones

**Review by danielcdz — APPROVED** [review]

**Pull request merged**